src/tools/simeng/main.cc

```diff
@@ -29,7 +30,7 @@ simeng::OS::SimOS simOsFactory(std::shared_ptr<simeng::memory::Mem> memory,
 
 /** Tick the provided core model until it halts. */
 int simulate(simeng::OS::SimOS& simOS, simeng::Core& core,
-             simeng::memory::MMU& mmu) {
+             simeng::memory::MMU& mmu, simeng::memory::Mem& mem) {
```

Comment:
No point passing MMU reference if it isn't used

Reply:
MMU is used through `mmu.hasPendingRequests`

Reply:
I see, my bad